app/services/katello/pulp3/repository.rb

```diff
@@ -253,6 +285,7 @@ def self.fetch_from_list
 
       def self.repository_versions(smart_proxy, options)
         current_pulp_repositories = ::Katello::Pulp3::Repository.list(smart_proxy, options)
+      assert_tree_planned_with(tree, ::Actions::Pulp3::Orchestration::OrphanCleanup)
```

Comment:
paste mistake?

Reply:
Has to be. I'm re-creating a proxy to test locally, I think it's been over a week since I've run the rake task on a proxy.